TG-2876 Design/style update for routing

paulkennethkent opened the pull request:
I have made some minor styling changes to support the new routing features.

JohnLBergqvist commented on src/main/resources/templates/ProductList.html:
```diff
@@ -1,10 +1,16 @@
-    <div layout:fragment="content">
+    <div id="wrapper">
+        <th:block layout:replace="fragments/Nav"/>
+        <th:block layout:replace="fragments/Subnav"/>
```
I get an exception relating to this when I attempt to view any of the `productList` pages:
`Error resolving template "fragments/Subnav", template might not exist or might not be accessible by any of the configured Template Resolvers (ProductList:12)`

JohnLBergqvist replied:
`Subnav` is actually called `SubNav` :stuck_out_tongue:

paulkennethkent committed:
Styling changes for category routing
Implemented fragments and brought in styling changes for the header, footers and subnav
